includes/Assets.php

```diff
@@ -346,6 +343,13 @@ public function get_styles() {
             ],
         ];
 
+        // Scenario like there is already fontawesome library added by the theme with the setting user can control if he/she want to load the dokan fontawesome library or not.
+        if ( is_admin() || 'on' === dokan_get_option( 'enable_dokan_fontawesome', 'dokan_appearance', 'on' ) ) {
+            $styles['dokan-fontawesome'] = [
+                'src' => DOKAN_PLUGIN_ASSEST . '/vendors/font-awesome/font-awesome.min.css',
+            ];
+        }
+
```

Comment:
we need to remove `dokan-fontawesome` dependencies from our codebase. Otherwise, this might through a console warning.